src/app/static/src/app/apiService.ts

```diff
@@ -61,6 +68,11 @@ export class APIService<S extends string, E extends string> implements API<S, E>
 
     private _onSuccess: ((success: Response) => void) | null = null;
 
+    freezeResponse = () => {
```

Comment:
We were freezing everything by default, but actually this is unnecessary and made updating the form options a pain (it's much easier to mutate the incoming object by just adding existing values, than to mutate the old object or create a new one.)